mandatory/2-function-creation.js

```diff
@@ -45,7 +80,12 @@ function canRegister(age) {}
   )
 */
 
-function countReverse(number) {}
+function countReverse(number) {
+  while (number >0){
+    console.log(number);
+  };
+  number=number-1;
+};
```

Comment:
Good effort but are you sure this is correct? When you ran this code locally did it work for you correctly? 
Look closely what you are trying to do with `number` parameter. Current code will result in never ending loop and will never complete is there a better way? 


Hint: Have a look at your `printOddNumbers` function.